snowflake-py-pulumi-search-export/__main__.py

```diff
@@ -0,0 +1,352 @@
+"""A Python Pulumi program"""
+
+import pulumi
+import pulumi_aws as aws
+import pulumi_command as command
+import pulumi_snowflake as snowflake
+
+import datetime
+import json
+import os
+
+bucket = aws.s3.Bucket("pulumi-search-export")
+
+config = pulumi.Config()
+# NOTE: In a production scenario, for security reasons this should should call
+# `config.require_secret()`, use a Pulumi access token specifically designated
+# for this export process, and not fall back to an environment variable:
+pulumi_access_token = config.get_secret(
```

Comment:
Preview fails without a `PULUMI_ACCESS_TOKEN`. Should we document this in the README as a pre-req?

Reply:
Yes. Will fix.